src/common_audio/vad/webrtc_vad.c

```diff
@@ -24,7 +24,9 @@ static const int kMaxFrameLengthMs = 30;
 
 VadInst* WebRtcVad_Create(void) {
   VadInstT* self = (VadInstT*)malloc(sizeof(VadInstT));
-
+  if (self == NULL) {
```

Comment:
Hmm. This should either be reported upstream (to Mozilla and/or Google) or we should update the WebRTC code from upstream... no longer sure where it lives exactly